docs/03-tutorials/00-application-connectivity/ac-08-rotate-root-ca.md

```diff
@@ -134,7 +134,7 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
 2. Generate a new certificate using the key you generated and save it to the `new-ca.crt` file.
 
    ```bash
-   openssl req -new -key new-ca.key -x509 -sha256 -days {EXPIRATION_DAYS} -nodes -out new-ca.crt
+   openssl req -new -key new-ca.key -x509 -sha256 -days {TTL_DAYS} -nodes -out new-ca.crt
```

Comment:
This tutorial could also be changed to use environment variables instead of placeholders.

Reply:
**TOCONSIDER**/**TODO**

Reply:
We no longer have this flow.